app/values/scanned_resource_pdf/renderer.rb

```diff
@@ -67,7 +67,7 @@ def metadata_hash
             hsh[entry["label"].to_sym] = Array(entry["value"]).join(", ")
           end
         result[:Title] = Array(scanned_resource.title).join(", ") if scanned_resource.title
-        result[:Description] = Array(scanned_resource.description).join(", ") if scanned_resource.description
+        result[:Description] = scanned_resource.description.join(", ") if scanned_resource.description
```

Comment:
I'm not sure this change is necessary. `Array([1,2,3])` leaves the array unchanged, while ensuring the description field returns an array.